frontend/src/views/Shares/ShareView.js

```diff
@@ -157,6 +161,28 @@ function ShareViewHeader(props) {
     }
     setRejecting(false);
   };
+  const revoke = async () => {
+    setRevoking(true);
+    const response = await client.mutate(
+      revokeAllShareObject({
+        shareUri: share.shareUri
+      })
+    );
+    if (!response.errors) {
+      enqueueSnackbar('All items if share request revoked', {
+        anchorOrigin: {
+          horizontal: 'right',
+          vertical: 'top'
+        },
+        variant: 'success'
+      });
+      await fetchItems();
+      await fetchItem();
+    } else {
+      dispatch({ type: SET_ERROR, error: response.errors[0].message });
+    }
+    setRevoking(false);
+  };
```

Comment:
Regarding Lines 197-212: seems like the "shares" breadcrumb is there twice by mistake

Reply:
Yeeep, thank you again